packages/manager/modules/dbaas-logs/src/logs/detail/osd/add/logs-osd-add.controller.js

```diff
@@ -0,0 +1,128 @@
+export default class LogsOsdAddModalCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $stateParams,
```

Comment:
**nitpick:** Avoid injecting `$stateParams` and use `resolve` method on the route definition to pass the params into the controller

Reply:
You are right, but it was already the case when this section was called "kibana". If it needs to be refactored, i suggest to do that in an other PR.